Authentication/controllers/displayPages.js

```diff
@@ -0,0 +1,19 @@
+const LandingPage = (req, res) => {
```

Comment:
you can remove this file

Reply:
and what about the part of the code who uses these functions. This code is necessary as if user after logging in tries to go to the login or signup page via url manipulation then he will not be able to do so and will redirected to the same page

Reply:
We will have a separate views for the login, signup etc
Not what you've included